release-blobstore.html.md.erb

```diff
@@ -39,6 +39,22 @@ blobstore:
 
 See [Configuring S3 release blobstore](s3-release-blobstore.html) for details and [S3 CLI Usage](https://github.com/pivotal-golang/s3cli#usage) for additional configuration options.
 
+## <a id="gcs-config"></a> GCS Configuration
+
+Google Cloud Storage can be used without S3 compatibility mode.
+
+`config/final.yml`:
+
+```yaml
+---
+blobstore:
+  provider: gcs
+  options:
+    bucket_name: <bucket_name>
+```
+
+You have to be logged in using gcloud to your Google account. There is no need for a `config/private.yml`.
```

Comment:
Hi. Thanks for proposing this.

I don't think this is accurate. If users want to specify a `json_key` option, the `gcloud` commands don't need to be installed.